Pipfile

```diff
@@ -13,4 +13,4 @@ mock = "*"
 tox = "*"
 
 [packages]
-serverlessrepo = {path = ".",extras = ["dev"],editable = true}
+serverlessrepo = {editable = true,extras = ["dev"],path = "."}
```

Comment:
Why change this?

Reply:
Pipenv changes to this order automatically when I re-install dependencies. Just a bit annoying to change back every time, so I decided to use Pipenv’s order...